p/i/index.php

```diff
@@ -42,14 +42,43 @@
 		}
 	}
 
+	$migrations_path = APP_PATH . '/migrations';
+	$applied_migrations_path = DATA_PATH . '/applied_migrations.txt';
+
+	// The next line is temporary: the migrate method expects the applied_migrations.txt
+	// file to exist. This is because the install script creates this file, so
+	// if it is missing, it means the application is not installed. But we
+	// should also take care of applications installed before the new
+	// migrations system (<1.16). Indeed, they are installed but the migrations
+	// version file doesn't exist. So for now, we continue to check if the
+	// application is installed with the do-install.txt file: if yes, we create
+	// the version file. Starting from version 1.17, all the installed systems
+	// will have the file and so we will be able to remove this temporary line
+	// and stop using the do-install.txt file to check if FRSS is already
+	// installed.
+	touch($applied_migrations_path);
```

Comment:
We might not need that. If the file does not exist, then we try to apply the migrations and finally create the file

Reply:
Yep, with your suggestion above, it might make sense :+1: I just need to take a bit of time to reconsider this first (I expect to work on it this week), this line had a high importance in my mind ^^

Reply:
So indeed, this line has its importance. Well it might not immediately, but I prefer to keep the current design and to make the Migrator failing if the file doesn't exist since the install process will have to start if `applied_migrations.txt` doesn't exist in a later version of FRSS.